.jazzy.yaml

```diff
@@ -0,0 +1,7 @@
+author: Alamofire Software Foundation
+author_url: http://alamofire.org/
+github_url: https://github.com/Alamofire/Alamofire
+module: Alamofire
+output: docs
```

Comment:
After more investigation, it looks like it has to go in the `master` branch in a `docs` folder to work seamlessly with GitHub Pages. We can sort out where we put everything else in another PR. Might not really be that big of a deal to have a `docs` directly full of Jazzy docs and then a `Documentation` folder with everything else nested.